### DISCUSSION TITLE
Looking into the openaddresses.io project

### DISCUSSION THREAD

**Discussion by johnwgoodin** [organicmaps/organicmaps#4191] [Ideas]
I heard about the project openaddresses.io that's designed for finding addresses. I really don't know a whole bunch about it. Could it be incorporated into Organic Maps as an alternate way of finding addresses offline?

Would adding integration of its features be a good feature request for organic Maps in the future? Could adding it make searching easier and more intuitive?

**Comment by johnwgoodin** [comment]
You can get information about the project at https://openaddresses.io

**Reply by biodranik** [reply]
There was already an issue about it. The main question is how to integrate/mix sideloaded address data with OpenStreetMap.org addresses 1) for compact and fast offline usage 2) to avoid breaking some scenarios (e.g. editing the address in OM/OpenStreetMap.org)

**Reply by johnwgoodin** [reply]
I was wondering if organic Maps had ever supported the idea of its using and supporting add ons. Could a add on be created for open addresses to have it setup as an alternative way of accessing addresses databases. I would be fine with using such a add on just so long as it didn't create any problems with the built in OpenStreetMaps databases. Say you had problems using OpenStreetMaps databases to locate a place or address. You could pop a switch to temporary activate the plug-in and port your search through it instead.

**Reply by biodranik** [reply]
The app should be just "usable". No annoying popups. It is possible to mix different sources of data, but it requires some work and some conflicts resolution approaches.
